00002_arithmeticOperations.js

```diff
@@ -0,0 +1,60 @@
+function ArithmeticAdd(num1, num2)
+{
+    var sum = 10 + 20;
+    console.log("sum is " + sum);
+}
+ArithmeticAdd();
+function ArithmeticSub(num3, num4)
+{
+    var difference = 20 - 10;
+    console.log("difference is " + difference);
+}
+ArithmeticSub();
+function ArithmeticProduct(num5, num6)
+{
+    var product = 10 * 10;
+    console.log("product is " + product);
+}
+ArithmeticProduct();
+function ArithmeticQuotient(num7, num8)
+{
+    var quotient = 20 / 10;
+    console.log("quotient is " + quotient);
+}
+ArithmeticQuotient();
+function ArithmeticModulus(num9, num10)
+{
+    var modulus = 45 % 23;
+    console.log("modulus is " + modulus);
+}
+ArithmeticModulus();
+function ArithmeticLess(num11, num12)
+{
+    var num11 = 10;
+    var num12 = 8;
+    if (num12 < num11)
+    {
+        console.log("Num12 is less than num11");
+    }
+}
+ArithmeticLess();
+function ArithmeticGreater(num13, num14)
+{
+    var num13 = 35;
+    var num14 = 40;
+    if (num14 > num13)
+    {
+        console.log("num14 is greater than num13")
+    }
+}
+ArithmeticGreater();
+function ArithmeticEqual(num15, num16)
```

Comment:
convert this to ES6 style 
arithmeticEqual = (num1, num2) = {
}

Reply:
Please change it in all other functions

Reply:
Converted this to ES6 style. Can you please review it.

Reply:
PR updated. Can yoy please review.